exonum/src/api/public/system.rs

```diff
@@ -19,6 +19,7 @@ use serde_json;
 use api::Api;
 use blockchain::{Blockchain, Schema, SharedNodeState};
 use helpers::user_agent;
+use std::net::SocketAddr;
```

Comment:
Standard imports should be placed after third-party usings, but before inner ones.